operators/endpointmetrics/cmd/cmo-config-revert/main.go

```diff
@@ -20,6 +21,11 @@ import (
 )
 
 func main() {
+	//set env as CMO_SCRIPT_MODE to enable the script mode
+	err := os.Setenv("CMO_SCRIPT_MODE", "true")
```

Comment:
Not a big fan of using env variables like this. Not sure if we could do something else?